Bomb.py

```diff
@@ -0,0 +1,39 @@
+import pygame
+
+class Bomb:
+    def __init__(self, x, y, power):
+        self.x = x
+        self.y = y
+        self.power = power
+        self.timer = 0
+
+    def __del__(self):
+        pass
+
+    def plant(self, win):
+        file = 0
+        if self.timer <= 40:
+            file = 'assets/bomb/bomb1.png'
+        elif self.timer <= 80:
+            file = 'assets/bomb/bomb2.png'
+        elif self.timer <= 120:
+            file = 'assets/bomb/bomb3.png'
+        elif self.timer <= 160:
+            file = 'assets/bomb/fire.png'
+        else:
+            return
+
+        img = pygame.image.load(file)
+        win.blit(img, (self.x, self.y))
+
+        if self.timer > 120:
+            for i in range(0, self.power):
+                win.blit(img, (self.x, self.y + 60 * i))
+                win.blit(img, (self.x, self.y - 60 * i))
+                win.blit(img, (self.x + 60 * i, self.y))
+                win.blit(img, (self.x - 60 * i, self.y))
```

Comment:
I would replace "60" with constant value

Reply:
Same with this `120`

Reply:
I think that it would be the best to extract this `60` to a constant value that is dependent on the FPS value. You could create a file `constants.py` where we could store such constants:
```py 
> constants.py

FPS = 60
```

Then you would change this `60` value to `SEC` where `SEC = FPS`

This enables the possibility of changing the fps values to be independent of the game.

Reply:
I changed these values into constants, but they weren't applying to FPS but height and width of the image.